### DISCUSSION TITLE
GUI problem, help

### DISCUSSION THREAD

**Discussion by weimath** [connectomicslab/connectomemapper3#68] [Q&A] [answered]
when i  input 'cmpbidsappmanager ', I could get the  complete interface.

![image](https://user-images.githubusercontent.com/16069593/103741341-d6aabd00-5033-11eb-9511-b9e978aaba37.png)

then

![image](https://user-images.githubusercontent.com/16069593/103741399-f2ae5e80-5033-11eb-9f27-c3faf381f2fa.png)

**Comment by weimath** [comment]
![QQ截图20210106153118](https://user-images.githubusercontent.com/16069593/103741709-84b66700-5034-11eb-9d56-b42aacee96b1.png)

**Comment by sebastientourbier** [comment, marked as answer]
> Have you selected your BIDS dataset directory? If not, this is normal.
> Just click on File -> Load BIDS Dataset...
> Then select your BIDS dataset directory and click OK.
> You should be all set.
> 
> Best
> Sebastien

From the Google Groups "cmtk-users" group discussion at https://groups.google.com/d/msgid/cmtk-users/20dc4032-5438-4946-ad62-36f799b3afc6n%40googlegroups.com.
